Yoast/YoastSEO.js
# Refactors passive voice assessment

## Summary

This PR can be summarized in the following changelog entry:
* Refactors passive voice assessment to improve language coverage.

## Relevant technical choices:

The passive voice assessment was refactored so that analysis of languages with both periphrastic and morphological passive voice marking is made possible. Additionally, functionality for Russian passive voice assessment is added. For clarity, passive voice analysis for Russian will only be made possible in the `feature/russian` branch and will be merged later.

## Test instructions

This PR can be tested by following these steps:

- Check out the branch.

- Use the browserified example. Don't forget to run a grunt build:js.

- Set the locale to en_EN, es_ES, fr_FR, or de_DE.

- Use examples from passive voice specs or similar.


Fixes #1456
